pyproject.toml

```diff
@@ -67,7 +67,7 @@ pytest-watch = "*"
 pytest-flask = "*"
 flake8 = "*"
 bumpversion = "*"
-black = "24.*"
+black = "25.*"
```

Comment:
I'm a little surprised if this didn't change anything in our codebase?

Reply:
But so it seems. Even when ran with `--preview` option, black does not detect anything to change (although in one of my helper scripts the `--preview` option does find some format change).